docs/manual/common/guide/devmode/CassandraServer.md

```diff
@@ -79,7 +75,9 @@ cassandra-snapshot-store.keyspace=users
 lagom.persistence.read-side.cassandra.keyspace=users
 ```
 
-Note that the keyspace values provided via the `application.conf` will always win over any keyspace name that may be set in the build.
+Note that the keyspace values provided via the `application.conf` will always win over any keyspace name that may be set in the build. For that reason, overriding the keyspace in the build is deprecated, and will be removed in a future version of Lagom.
```

Comment:
How about: "Note that Cassandra uses keyspace values from the application.conf file instead of any you might define in the build. For that reason ..."